manifests/4.4/cluster-loggings.crd.yaml

```diff
@@ -120,6 +120,12 @@ spec:
                       - "ZeroRedundancy"
                   required:
                   - nodeCount
+                retentionPolicy:
+                  type: object
+                  properties:
+                    maxAge:
```

Comment:
this should actually be something that includes the logsource type:
`logs.app', `logs.infra`, `logs.audit` which have a property of `maxAge`

Ref: https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#clusterlogging-api